test/unit/index.test.ts

```diff
@@ -124,6 +125,7 @@ const EXPECTED_EXPORTS = [
   'ServerType',
   'SrvPollingEvent',
   'Timestamp',
+  'TokenCache',
```

Comment:
This should be removed when you move this to a type only export.

Reply:
Removed

Reply:
We are still exporting these, we need to make sure we use "export type"